Dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM centos:7.4.1708
-ARG BINARY=./mobile-developer-console
+FROM registry.access.redhat.com/rhoar-nodejs/nodejs-10
```

Comment:
registry.access.redhat.com is due to go away at some point (either this year or next year I'd say) so maybe we shouldn't use it? Also, if we intend to build this image and host it somewhere like quay.io, I'm not sure if we'd be allowed (it depends on the base of this image I think). I'm not sure what alternative to use instead, do they have something on quay.io?

Reply:
I don't really know TBH, couldn't find any other documents. Maybe we should just use the community nodejs image?

Reply:
That would be a better option I think. Alternatively there's this image which is the upstream version of the RHSCL Node.js 10 image I believe: https://hub.docker.com/r/centos/nodejs-10-centos7

I wasn't able to find any upstream image from RHOAR, but I might just not be looking in the right places. I'll see if I can find someone to ask